src/Speckle.Objects/Data/Civil3dObject.cs

```diff
@@ -20,13 +20,15 @@ public class CivilObject : Base, ICivilObject
   /// Children objects, eg profiles, this civil entity may contain.
   /// </summary>
   [DetachProperty]
-  public required List<CivilObject> elements { get; set; }
+  public required List<Base> elements { get; set; }
 
   public required List<Base> displayValue { get; set; }
 
+  public required Dictionary<string, object?> properties { get; set; }
+
   public required string units { get; set; }
 
-  IReadOnlyList<ICivilObject> ICivilObject.elements => elements;
+  IReadOnlyList<Base> ICivilObject.elements => elements;
```

Comment:
We should double check that the serializer doesn't accidently try and grab this property @adamhathcock 

We can set these as `[JsonIgnore]`, but I'd like to understand if there's something else that's nicely guaranteeing that the "hidden" elements prop (explicit interface implementation) is not ever prefered over the "real" `.elements` property.